RELEASE.md

```diff
@@ -25,11 +36,65 @@ The Dragonfly project maintains a dynamic development cycle, addressing both exi
 
 Given the complexity of significant features, their full integration often spans multiple release cycles. As a release nears completion, it's not uncommon for complex features to be rescheduled for the subsequent milestone. This approach ensures each feature meets the rigorous requirements of the project's feature lifecycle, maintaining the high quality standards of the Dragonfly project.
 
-## Release note
+## Release Lifecycle
+
+The release lifecycle is a structured process that ensures the quality and stability of Dragonfly releases. It consists of several key phases:
+
+1. **Stabilization Phase**: This phase concentrates on fixing bugs and ensuring the stability of the upcoming release. It spans the 4 weeks
+   leading up to the Release Date, starting with the Code Freeze. The Code Freeze takes place 4 weeks before the scheduled Release Date,
+   at which point a dedicated release branch is created from the main branch. During this stabilization period, only bug fixes are merged into
+   the release branch, while new features continue to be integrated into the main branch.
+
+2. **Maintenance Phase**: During this phase, the release is maintained, and critical bugs are addressed. This phase lasts for 6 months,
+   starting from the current version Release date and continuing until the next version is released. However, maintenance support is provided
+   only for the immediately preceding version. For example, release-2.1 branch is kept open for bug fixes, and the main branch is used for
+   v2.2 features and improvements.
+
+### Stabilization Phase
```

Comment:
This section seems duplicated with above **Stabilization Phase** described in the Release Lifecycle.